packages/manager/src/features/Account/constants.ts

```diff
@@ -9,6 +9,7 @@ export const grantTypeMap = {
   firewall: 'Firewalls',
   image: 'Images',
   linode: 'Linodes',
+  lkeCluster: 'Clusters',
```

Comment:
I think this code comment will bring some clarity that it's mainly used to improve UX
```suggestion
  lkeCluster: 'Clusters', // Note: Not included in the user's grants returned from the API.
```

Reply:
Nit: Can we also rename this to `lke: LKE Clusters`?